cmd/fsi_integration_test.go

```diff
@@ -758,19 +758,19 @@ func TestGetPreviousVersionExplicitPath(t *testing.T) {
 	defer run.Delete()
 
 	// First version has only a body
-	output := run.MustExec(t, "qri save --body=testdata/movies/body_two.json me/get_ver")
+	output := run.MustExecCombinedOutErr(t, "qri save --body=testdata/movies/body_two.json me/get_ver")
```

Comment:
Also not even sure this is the correct approach, but got me over the fence pretty quickly.

Reply:
yep I think this is correct. I had to do the same in my branch